tests/plugins/test_ingest_website.py

```
@@ -166,6 +166,258 @@ def test_strips_scripts(self):
+        event = make_ingest_website()
+
+        with patch("plugins.ingest_website.plugin.crawl", return_value=pages), \
+             patch("plugins.ingest_website.plugin.IngestEngine") as mock_engine:
+            import asyncio
+            mock_engine.return_value.run = lambda *a, **kw: asyncio.coroutine(
+                lambda: MagicMock(success=True, errors=[])
+            )()
+            await plugin.handle(event)
+
+        # IngestEngine.run receives the document list — check it doesn't include empty docs
+        run_call = mock_engine.return_value.run
```

Flake8 (7.3.0)

[error] local variable 'run_call' is assigned to but never used

(F841)